---
layout: page
title: Identifying Shared Features Among Resistance Mechanisms to Help Predict Effective Combination Therapies for Individual Patients
---

Targeted therapies extend many cancer patient's lives, but are limited in efficacy to a subset of patients and by the development of resistance. Enormous efforts undertaken to identify mechanisms of resistance have uncovered numerous changes involving gene expression, post-translational regulation, and even tumor-extrinsic factors such as host-derived growth factors. Combination therapy can effectively combat resistance, but requires accurate identification of the relevant resistance mechanism. Precision therapy must account for many genetic and non-genetic intrinsic and adaptive resistance mechanisms if it will accurately select these combinations.

Rather than focus on single molecular changes causing resistance, we are studying sets of these changes to reveal the essential commonalities. Methods studying the signaling network changes leading to resistance to date have largely focused on two approaches: (1) paired molecular and response measurements across large panels of cell lines, or (2) screening-based platforms in which a large panel of expression changes are assessed for effect on resistance. These are complementary and informative but limited in the information provided. In the former case, one is limited to the variation within the cell line panel, with the ultimate resistance mechanism often unknown. Further, widespread genetic variation between cell lines serves as "noise" diluting out the "signal" of exactly which signaling changes lead to resistance, and may not be representative of resistance derived from sources such as the tumor microenvironment. In the latter case, sparse accounting for the molecular changes present with each intervention limits the commonalities that can be identified. Pinpointing the necessary and sufficient signaling changes for resistance mechanisms is essential for a clearer picture of what to measure and target in individual patient's tumors. For example, if bypass resistance to EGFR inhibitors in lung adenocarcinoma mediated by cMET, AXL, and FGFR1 all rely on Erk *or* JNK activation mediated by Grb2 *or* CrkL, then we can use this logic to identify precise treatment combinations. One might, with this knowledge, target AXL if activated cMET and FGFR1 are not present, or CrkL if Grb2 is not active. Realizing precision medicine in this fashion, however, requires identification both of the essential molecular events for resistance and mapping between the various signaling layers.

> <img src="/public/images/resistanceConcept.svg" width="600px" />  
> In RTK-driven tumors, signals are transduced from the receptor to various kinases. Upon blocking the original cancer driver, resistance can be conferred by an untargeted receptor. Some receptors, however, do not provide essential resistance signals. By identifying similarities and differences of signaling from each receptor, we will be able to identify measurements pinpointing the relevant receptor causing resistance.

We are currently applying an alternative approach as part of my NIH Director's Early Independence Award to understand resistance to targeted kinase inhibition mediated by alternative, non-targeted receptor tyrosine kinases (RTKs)—so called bypass resistance. A central approach we take for this work is to use cells sensitive to an RTK inhibitor and then rescue them to varying extents with a panel of growth factors, cytokines and/or transient expression of selected genes. We then quantify the apoptosis/proliferative response of the cells in the presence of the inhibitor in each case, paired with multiplexed measurement of the signaling network changes with each intervention. Using multivariate modeling, we identify the molecular features that predict resistance, then experimentally verify these implicated events. This serves as an example for the benefit of close integration between the experimental and modeling efforts—through matched genetic backgrounds with very little time for subclonal selection, this approach provides optimal measurements for the modeling question at hand.

This framework has already led us to identify that pathways beyond just canonical markers such as Erk/Akt are important for identifying whether an RTK can make cells resistant to EGFR- and HER2-targeted therapies. Namely, JNK activation is essential for predicting resistance, and modulating JNK activity in concert with other pathways can influence resistance development. Having published this initial work as a proof of concept, we are now applying this to identify drug combinations and study tumor heterogeneity. Lung cancer cells display dynamic, heterogeneous activation of Erk and JNK, and so we are mapping the multi-pathway single cell variation in activation as a potential mechanism of tumor persistence and adaptive resistance. With collaborators, we are also testing whether receptor-proximal measurements can identify which RTK is driving disease, and if these measurements in turn can predict optimal treatment combinations for individual xenograft tumors.

Identifying commonalities among the many expression and signaling changes that cause cells to become resistant is a critical compliment to the functional genetics studies that have now globally mapped these changes. Using this approach, we will be able to determine whether cells reactivate the same downstream kinases or alternatively rely on fundamental changes in pathway activation dependency for a wider panel of molecular network changes driving resistance. This map will enable us to identify which combination therapies will be effective for individual tumors given measurements for which kinases are active.
